Cairo Desktop/Cairo Desktop/Services/CairoApplicationThemeService.cs

```diff
@@ -26,15 +31,15 @@ public sealed class CairoApplicationThemeService : IThemeService, IDisposable
         };
         private readonly ILogger<CairoApplicationThemeService> _logger;
         private readonly Settings _settings;
+        private readonly RegistryEventWatcher _watcher;
 
         public CairoApplicationThemeService(ILogger<CairoApplicationThemeService> logger, Settings settings)
         {
             _logger = logger;
             _settings = settings;
-
             MigrateSettings();
-
             _settings.PropertyChanged += Settings_PropertyChanged;
+            _watcher = new RegistryEventWatcher(this);
```

Comment:
Because this is instantiated here, the registry watcher always starts (its constructor calls `WatchRegistry(true)`). I'd prefer to avoid the overhead of initializing WMI when we don't need it, so maybe instead of instantiating the watcher here, we only instantiate it when FollowThemeColors=true?

Reply:
The object has been updated with explicit methods to start/stop looking for events. It won't start in the constructor, anymore